### PR TITLE
fedekunze/1464 li governance txs

### LINKED ISSUE #1464: Create LiTransaction for governance
Description:

Required to display governance txs on the Transaction Page

- [x] TmUI Components
- [ ] Integrate to voyager wallet

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #1464

_Description:_

- Adds governance txs to the tx history page

❤️ Thank you!

---

* [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
* [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

**Comment by fedekunze** [comment]
blocked by release of TmUI

**Comment by faboweb** [comment]
released 0.3.1

File: app/src/renderer/routes.js [hunk 1]
```diff
@@ -12,19 +12,22 @@ export default [
     path: `/governance`,
     name: `Governance`,
     component: governance(`Governance`),
-    redirect: `proposals/`,
+    redirect: `/governance/proposals`,
     children: [
       {
-        path: `/proposals`,
```
[the remark this answers is not in the document]
Comment (fedekunze) [reply app/src/renderer/routes.js]: yeah

File: app/src/renderer/vuex/modules/transactions.js [hunk 2]
```diff
@@ -47,10 +51,13 @@ export default ({ node }) => {
...
       commit(`setStakingTxs`, stakingTxs)
 
+      const governanceTxs = await dispatch(`getTx`, `governance`)
+      commit(`setGovernanceTxs`, governanceTxs)
+
       const walletTxs = await dispatch(`getTx`, `wallet`)
       commit(`setWalletTxs`, walletTxs)
 
-      const allTxs = stakingTxs.concat(walletTxs)
+      const allTxs = stakingTxs.concat(governanceTxs.concat(walletTxs))
```
[the remark this answers is not in the document]
Comment (fedekunze) [reply app/src/renderer/vuex/modules/transactions.js]: yeah, sadly `+` is not supported for array concat: 
https://stackoverflow.com/questions/7124884/why-is-1-2-3-4-1-23-4-in-javascript

**Pull request merged**